Automattic/jetpack
# Gist Shortcode: Remove jQuery Dependency

Part of #16409

#### Changes proposed in this Pull Request:

* Removes the use of jQuery for the Gist shortcode

#### Jetpack product discussion



* N/A

#### Does this pull request change what data or activity we track or use?



Nope.

#### Testing instructions:






Try using a Gist shortcode with different **ts** parameters. For example:

```
[gist jeherve/57cc50246aab776e110060926a2face2] (ts defaults to 8)
[gist ts="2" jeherve/57cc50246aab776e110060926a2face2]
```

Verify that the Gist appears with the additional styling, and the `tab-size` style is also inserted.

<img width="661" alt="Screenshot 2020-08-04 at 11 06 51" src="https://user-images.githubusercontent.com/43215253/89282876-1c59f880-d644-11ea-92ed-d324edc03f8f.png">


#### Proposed changelog entry for your changes:

* Shortcodes: remove jQuery dependancy from Gist shortcode.
